draft-nuttycom-funding-allocation.rst

```diff
@@ -0,0 +1,609 @@
+::
+
+  ZIP: Unassigned
+  Title: Block Reward Allocation for Non-Direct Development Funding
+  Owners: Kris Nuttycombe <kris@nutty.land>
+          Jason McGee <aquietinvestor@gmail.com>
+  Original-Authors: Skylar Saveland <skylar@free2z.com>
+  Credits: Daira-Emma Hopwood
+           Jack Grigg
+           @Peacemonger (Zcash Forum)
+  Status: Draft
+  Category: Consensus
+  Created: 2024-07-03
+  License: MIT
+  Pull-Request: <https://github.com/zcash/zips/pull/866>
+
+Terminology
+===========
+
+The key words "MUST", "REQUIRED", "MUST NOT", "SHOULD", and "MAY" in this
```

Comment:
Nit: REQUIRED isn't used. (There is a lowercase usage but it doesn't appear to be intended as a conformance term.)